app/src/main/java/com/sdp/movemeet/LoginActivity.java

```diff
@@ -31,33 +31,33 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_login);
 
-        mEmail = findViewById(R.id.edit_text_email);
-        mPassword = findViewById(R.id.edit_text_password);
+        email = findViewById(R.id.edit_text_email);
+        password = findViewById(R.id.edit_text_password);
         progressBar = findViewById(R.id.activity_login_progress_bar);
         fAuth = FirebaseAuth.getInstance();
-        mLoginBtn = findViewById(R.id.button_login);
-        mCreateBtn = findViewById(R.id.text_view_create_account);
+        loginBtn = findViewById(R.id.button_login);
+        createBtn = findViewById(R.id.text_view_create_account);
 
-        mLoginBtn.setOnClickListener(new View.OnClickListener() {
+        loginBtn.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
 
                 // When the user clicks on the "LOGIN" button, we first validate his data
-                String email = mEmail.getText().toString().trim();
```

Comment:
Why were you able to directly call the attribute before, but not anymore ? Can't you directly write "email.getText().toString().trim()" ?

Reply:
You're completely right! By refactoring the name of some variables to match our convention, Android Studio may have rewritten variables this way. I corrected this by hand in the meantime!